src/strands/types/a2a.py

```diff
@@ -0,0 +1,26 @@
+"""Additional A2A types."""
+
+from typing import TypeAlias
+
+from a2a.types import Message, Task, TaskArtifactUpdateEvent, TaskStatusUpdateEvent
+
+from ._events import TypedEvent
+
+A2AResponse: TypeAlias = tuple[Task, TaskStatusUpdateEvent | TaskArtifactUpdateEvent | None] | Message
+
+
+class A2AStreamEvent(TypedEvent):
+    """Event that wraps streamed A2A types."""
```

Comment:
Is this emitted for every single update that we get from the remote server?  Even final updates etc?

If so, can we document that